cli_test2.py

```diff
@@ -0,0 +1,49 @@
+'''
+Testing file (TY)
+ - I likely will be writing my own test files
+    since my TextEdit crashes sometimes when loading your code... orz
+'''
+
+#==================================================#
+'''Import'''
+
+import toml
+from logic.common import file_utils as fu
+#	from logic.common import level_reader as lr
+#	from logic.common import level_writer as lw
+#	from logic.common import image_exporter as ie
+from logic.test import test_logic as logic
+
+
+#==================================================#
+
+# I put main() inside a function
+#  in order to comment out single lines without affecting indentation
```

Comment:
I prefer

```
def main():
#	function1()
	function2()
main()
```

over

```
# function1()
function2()
```

because they still "look" aligned between normal and disabled lines.